third_party/bigframes_vendored/pandas/core/frame.py

```diff
@@ -1662,6 +1664,15 @@ def dropna(
             <BLANKLINE>
             [3 rows x 3 columns]
 
+        Define in which columns to look for missing values.
```

Comment:
From doctest.
@@ -1,5 +1,5 @@
    -       name        toy       born
    -1    Batman  Batmobile 1940-04-25
    -2  Catwoman   Bullwhip        NaT
    +       name        toy        born
    +1    Batman  Batmobile  1940-04-25
    +2  Catwoman   Bullwhip        <NA>
     <BLANKLINE>
     [2 rows x 3 columns]

Reply:
Thanks, fixed! The diff got a little big because I pulled in https://github.com/googleapis/python-bigquery-dataframes/pull/987. It should be easier to review again once that is merged.